### DISCUSSION TITLE
No google icon?

### DISCUSSION THREAD

**Discussion by nimeshmaharjan1** [lucide-icons/lucide#1370] [Q&A] [answered]
I cant seem to find the google icon

**Comment by uixray** [comment]
I don’t know how to do this, but can I suggest this icon myself?

**Comment by karsa-mistmere** [comment, marked as answer]
Brand icons are deprecated, no new brand icons will be added, existing ones will eventually be removed: #94 #670
We advise you to use simpleicons.org
